invenio_app_rdm/translations/messages.pot

```diff
@@ -1,57 +1,108 @@
 # Translations template for invenio-app-rdm.
-# Copyright (C) 2019 CERN
+# Copyright (C) 2020 CERN
```

Comment:
minor: you can add your institution's copyright here @baldemir since it is thanks to you we have translations setup correctly.